### ISSUE TITLE
Not seeing the Magnum in the Demos ?

### ISSUE THREAD

**Issue by synchromesh62** [TheGameCreators/GameGuruRepo#3122]
**Not seeing the Magnum in the Demos ?**
Have you renamed it and forgot to re-add it or is it just me ?

**Comment by LeeBamberTGC** [comment]
@synchromesh62 Which demo, thanks :)

**Comment by synchromesh62** [comment]
> @synchromesh62 Which demo, thanks :)

Hi Lee
I first noticed it with mine  ( trapped ) But could it be just me ?
I was hoping to get a **Same For Me** response from others.
I can try a reinstall if its ok for you and the team.

**Comment by LeeBamberTGC** [comment]
@synchromesh62 I did fix an issue of the magnum and ammo being from the showcase folder, which is now devoid of weapons. I then added the weapons back in the latest build from the weapons folder, the latest versions of them. Maybe delete trapped from yoiur writable folder so you get the latest version of the level too :)

**Comment by synchromesh62** [comment]
> @synchromesh62 I did fix an issue of the magnum and ammo being from the showcase folder, which is now devoid of weapons. I then added the weapons back in the latest build from the weapons folder, the latest versions of them. Maybe delete trapped from yoiur writable folder so you get the latest version of the level too :)

Ok i will try that :)
I did notice this log file in with my mapbank so maybe thats what i need to do. 
**MISSING MEDIA:
entitybank\Showcase\Mk19T.fpe=replaceentitybank\Showcase\Mk19T.fpe**

**Comment by LeeBamberTGC** [comment]
@synchromesh62 Yes the plan will be to replace the showcase folder assets with a redirection system so the showcase can still host objects of interest, but they reside elsewhere in their proper place in the object library.  I will modify this issue into that task, saves some time :)
